lib/advanced-open-file-view.coffee

```diff
@@ -244,6 +245,7 @@ class AdvancedFileView extends View
       newPath = ''
 
     @miniEditor.setText(newPath)
+    @miniEditor.getModel().scrollToCursorPosition()
```

Comment:
Neato, never heard of this method before! Much easier than the old scroll hack thing I used before, I decided to [replace it](https://github.com/Osmose/advanced-open-file/commit/6fdd867f509aaf68acbd23617558eddfa83d8875).